src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/CortexTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Commands.Network.Test.ScenarioTests
+{
+    using System;
+    using Microsoft.WindowsAzure.Commands.ScenarioTest;
+    using Xunit;
+    using Microsoft.Azure.ServiceManagemenet.Common.Models;
+    using Xunit.Abstractions;
+
+    public class CortexTests : Microsoft.WindowsAzure.Commands.Test.Utilities.Common.RMTestBase
+    {
+        public XunitTracingInterceptor _logger;
+
+        public CortexTests(ITestOutputHelper output)
+        {
+            _logger = new XunitTracingInterceptor(output);
+            XunitTracingInterceptor.AddToContext(new XunitTracingInterceptor(output));
+        }
+
+        [Fact(Skip = "Needs ignite changes to be rolled out to all NRPs.")]
```

Comment:
This test is dependent on Ignite NRP rolling out to all regions. I can unblock this test, but it may be noted that this will fail till NRP rollout completes and the ARM manifest is updated. We can discuss and take the necessary changes for this.

Reply:
I have complete session record for this test. I can check that in if necessary.